src/API/Base.php

```diff
@@ -284,6 +284,12 @@ public static function get_account_info() {
 	}
 
 
+	public static function get_linked_businesses() {
+		$response = self::make_request( 'users/me/businesses', 'GET' );
+		return $response;
```

Comment:
`$response` is unnecessary here.

```suggestion
		return self::make_request( 'users/me/businesses', 'GET' );
```

Reply:
Removed here and on all other methods of this file